client/src/components/RelatedItemsAndOutfit/Carousel.jsx

```diff
@@ -32,18 +32,18 @@ const Carousel = ({ product, styles, related, reviews, carouselType }) => {
     // 2. Review data
     // 3. Default style picture URL
     const createdProductCards = [];
-    related.forEach(rel => {
+    relatedProducts.forEach(rel => {
       let totalRating = 0;
       let numberOfReviews = 0;
-      for (const ratingNumber in reviews.ratings) {
-        numberOfReviews += Number(reviews.ratings[ratingNumber]);
-        totalRating += ratingNumber * Number(reviews.ratings[ratingNumber]);
+      for (const ratingNumber in rel.reviewsMeta.ratings) {
+        numberOfReviews += Number(rel.reviewsMeta.ratings[ratingNumber]);
+        totalRating += ratingNumber * Number(rel.reviewsMeta.ratings[ratingNumber]);
       }
-      const defaultStyle = styles.results.filter(style => style['default?'])[0];
+      const defaultStyle = rel.styles.filter(style => style['default?'])[0] || rel.styles[0];
       const productCardInformation = {
-        ...product,
+        ...rel,
         rating: totalRating / numberOfReviews,
-        style: defaultStyle
+        defaultStyle: defaultStyle,
```

Comment:
Interesting choice. The trailing comma is a style choice I've seen since adding additional properties will only change one line.